fbw-a380x/src/wasm/systems/a380_systems_wasm/src/lib.rs

```diff
@@ -370,6 +370,7 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
     .provides_aircraft_variable("PAYLOAD STATION WEIGHT", "Pounds", 16)?
     .provides_aircraft_variable("PAYLOAD STATION WEIGHT", "Pounds", 17)?
     .provides_aircraft_variable("PAYLOAD STATION WEIGHT", "Pounds", 18)?
+    .provides_named_variable("AIRLINER_CRUISE_ALTITUDE")?
```

Comment:
I assume this is set by the FCU correct?

Reply:
Not the FCU, it's set manually by the pilot in the MFD / FMC.